### PR TITLE
[WIP] Improve version info

### PR DESCRIPTION
**Pull request by s-martin**
* save used git commit hash to version file  
* use saved git commit to determine if same version is already installed 

see #828

### CODE REVIEW

**Comment by MiczFlor** [comment]
Hi @s-martin 
the conflict reported here by github refers to the installation section for spotify. Not sure which one is the latest version. Looks like the code has moved on? Could you take a look?
I will be more present now, I promise :)

**Comment by s-martin** [comment]
I fixed the conflict.

The check, if same version is already installed is still missing, but we could add that later.

File: settings/version [hunk 1]
```diff
@@ -1 +1 @@
-2.0rc8
+2.0 %GIT_COMMIT%
```
Comment (ZyanKLee) [inline settings/version]: is this now the final version 2.0? Otherwise the `rc8` should in my opinion be kept in the version string.

Comment (s-martin) [reply settings/version]: As this goes into develop and 2.0 will be the next release we could just go for 2.0 except we expect another RC?

**Pull request merged**